VK/VK/OperationQueue/SaveToRealm.swift

```diff
@@ -0,0 +1,14 @@
+// SaveToRealm.swift
+// Copyright © RoadMap. All rights reserved.
+
+import RealmSwift
+
+/// Передача данных
```

Comment:
```suggestion
/// Передача данных
```
Не соответствует названию класса